dali/pipeline/data/buffer.h

```diff
@@ -433,11 +433,11 @@ class DLL_PUBLIC Buffer {
 };
 
 template <typename Backend>
-DLL_PUBLIC double Buffer<Backend>::growth_factor_ = 1.0;
+DLL_PUBLIC double Buffer<Backend>::growth_factor_ = 1.1;
```

Comment:
I have second thought about this change. I think it should go to a separate PR.

Reply:
Also, this change calls for all tests (L1, L2, L3).

Reply:
alright